### PR TITLE
INS-3078 edit api

### PR DESCRIPTION
**Pull request by fee1good**

### CODE REVIEW

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -1650,11 +1650,8 @@ paths:
               $ref: "#/components/schemas/ExportRequest"
       responses:
         200:
-          description: The export started. Returns the export file name.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Export"
+          description: The stage has been deleted
+          content: {}
```
Comment (aehrenthal) [inline sell/sell-api.yaml:1654]: The description does not seem right

**Review by aehrenthal — CHANGES_REQUESTED** [review]

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -1650,11 +1650,8 @@ paths:
               $ref: "#/components/schemas/ExportRequest"
       responses:
         200:
-          description: The export started. Returns the export file name.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Export"
+          description: The export started.
```
Comment (Zemidar) [inline sell/sell-api.yaml:1653]: Please add some information about sending email after finished export process

File: sell/sell-api.yaml [hunk 3]
```diff
@@ -1650,11 +1650,8 @@ paths:
...
+          content: {}
```
Comment (Zemidar) [inline sell/sell-api.yaml:1654]: image

**Review by Zemidar — CHANGES_REQUESTED** [review]

**Comment by fee1good** [comment]
Need approve pls for merge https://github.com/inperium-corp/inperium-server-sell/pull/979

**Review by Zemidar — APPROVED** [review]

**Comment by fee1good** [comment]
@RockMother review pls 
image

**Review by RockMother — APPROVED** [review]

**Comment by fee1good** [comment]
image

@aehrenthal review pls

File: sell/sell-api.yaml [hunk 4]
```diff
@@ -1649,12 +1649,8 @@ paths:
...
               $ref: "#/components/schemas/ExportRequest"
       responses:
-        200:
-          description: The export started. Returns the export file name.
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Export"
+        204:
+          description: The export started. Sending email link after finished export process.
```
Comment (aehrenthal) [inline sell/sell-api.yaml:1653]:
```suggestion
          description: The export started. You will receive a download link via email when the export is finished.
```

**Review by aehrenthal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**